app/controllers/requests_controller.rb

```diff
@@ -20,6 +20,18 @@ def create
       unless @request.save 
         render json: {error: "Request not saved"}, status: :bad_request
       end
+      PrivatePub.publish_to "/valet/new", :request => 
+      {
+        id:             @request.id,
+        name:           "#{@request.user.first_name} #{@request.user.last_name}",
+        picture:        @request.user.profile_picture,
+        transmission:   @request.user.is_manual,
+        phone:          @request.user.phone_number,
+        latitude:       @request.source_location.latitude,
+        longitude:      @request.source_location.longitude,
+        location:       @request.source_location.address,
+        type:           "pick_up"
+      }
```

Comment:
Is there a better way to display this information?  I tried to render it in a jbuilder but it just gave me an error and the docs for private pub are not very clear for what to do.  Maybe I could publish_to inside the jbuilder file?